### PR TITLE
declare a variable to instead the magic number

### PR DESCRIPTION
**Pull request by pengbinbin1**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by gy95** [comment]
need be `gofmt`

File: edge/pkg/eventbus/common/util/common.go [hunk 1]
```diff
@@ -16,7 +16,9 @@ import (
 
 var (
 	// TokenWaitTime to wait
-	TokenWaitTime = 120 * time.Second
+	TokenWaitTime      = 120 * time.Second
+	// Loop connect to wait
+	LoopConnectPeriord = 5 * time.Second
```
Comment (fisherxu) [inline edge/pkg/eventbus/common/util/common.go:21]: The file need to be go fmted :)

Comment (pengbinbin1) [reply edge/pkg/eventbus/common/util/common.go:21]: I have fmted it

Comment (pengbinbin1) [reply edge/pkg/eventbus/common/util/common.go:21]: @fisherxu

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**